src/main/java/sx/blah/discord/api/internal/DiscordWS.java

```diff
@@ -47,6 +48,11 @@
 
 public class DiscordWS extends WebSocketAdapter {
 
+	/**
+	 * The version of the Discord Gateway that is used by Discord4J.
+	 */
+	public static final String GATEWAY_VERSION = "6";
```

Comment:
Technically, it is possible to connect to the gateway with different version than the one used for REST. (At the moment, even, because D4J does not specify a version for REST, we use v3 for that and v5 for gateway). However, I doubt we will ever do that and it would be simple enough to change if we do for whatever reason. I'll combine them.